lib/package/commands.coffee

```diff
@@ -30,6 +30,9 @@ module.exports =
         @withInk ->
           boot()
           juno.runtime.evaluation.toggleMeta 'methods'
+      'julia-client:reset-workspace': =>
+        boot()
```

Comment:
Does it make sense to boot here? If the client has just been booted I would've expected calling `workspace()` to be redundant anyway – in which case `requireClient` makes more sense.